src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -95,9 +95,16 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
   useEffect(() => {
     const fetchData = async () => {
       setState({ loading: true });
+      const endDate = new Date(
+        new Date().getFullYear(),
+        new Date().getMonth(),
+        new Date().getDate() - 1,
+      );
```

Comment:
How does it work with Local/UTC dates?

Reply:
This is not working, I've forgotten datahub uses the time in the date.  😢

Reply:
@andresmoschini regarding Local and UTC times, it works just as `dateFrom` does.

Reply:
I am not sure to understand, do you have a screenshot of the request that webapp does against datahub?